### PR TITLE
removes bad field references

### PR DESCRIPTION
**Pull request by hlminh2000**

### CODE REVIEW

File: src/components/CohortBuilder/Categories/index.js [hunk 1]
```diff
@@ -45,23 +45,19 @@ const CATEGORY_FIELDS = {
...
+    'outcome.vital_status',
 
     // Available Data
     'available_data_types',
   ],
   quickSearch: [
     'available_data_types',
     'diagnoses.diagnosis',
-    // TODO - test both, see which one is the good one
-    // 'particpant.diagnosis_category',
```
Comment (hlminh2000) [inline src/components/CohortBuilder/Categories/index.js]: `particpant.diagnosis_category` is definitely not a field for participant :)

File: src/components/CohortBuilder/Categories/index.js [hunk 2]
```diff
@@ -45,23 +45,19 @@ const CATEGORY_FIELDS = {
     'diagnoses.diagnosis',
     'diagnoses.diagnosis_category',
     'family.family_compositions.composition',
-    // "Cannot query field "family__family_compositions__shared_phenotype_ids" on type "participantAggregations". Did you mean "family__family_compositions__shared_hpo_ids", "family__family_compositions__available_data_types", "family__family_compositions__composition", "family__family_compositions__family_members__kf_id", or "family__family_compositions__family_members__race"?"
-    // 'family.family_compositions.shared_phenotype_ids',
+    'family.family_compositions.shared_hpo_ids',
```
[the remark this answers is not in the document]
Comment (hlminh2000) [reply src/components/CohortBuilder/Categories/index.js]: unless performance is a concern with this field...

[the remark this answers is not in the document]
Comment (jberube) [reply src/components/CohortBuilder/Categories/index.js]: No, it was missing, but I think I was on prod at that moment.

File: src/components/CohortBuilder/Categories/index.js [hunk 3]
```diff
@@ -45,23 +45,19 @@ const CATEGORY_FIELDS = {
...
     // "Cannot query field "outcome__is_disease_related" on type "participantAggregations". Did you mean "outcome__disease_related", "outcome__external_id", or "outcome__vital_status"?"
     'outcome.disease_related',
     // 'phenotype.hpo_phenotype_observed_text',
     'diagnoses.source_text_tumor_location',
-    // "Invalid pivot field "outcome", not a nested field"
-    // 'outcome.vital_status',
+    'outcome.vital_status',
```
[the remark this answers is not in the document]
Comment (jberube) [reply src/components/CohortBuilder/Categories/index.js]: :+1:

**Pull request merged**